nimgen.nim

```diff
@@ -221,9 +221,10 @@ proc search(file: string): string =
         found = true
         break
     if not found:
-      echo "File doesn't exist: " & file
-      quit(1)
+      raise newException(Exception, "File doesn't exist: " & file)
 
+  # Only keep relative directory
+  result = result.replace(gProjectDir & $DirSep, "")
```

Comment:
Probably change to:

`return result.replace(re"[\\/]", $DirSep).replace(gProjectDir & $DirSep, "")`

This way, the second replace is guaranteed.

Reply:
Or perhaps use `multiReplace()`